.bin/go.mod

```diff
@@ -4,18 +4,29 @@ go 1.16
 
 replace github.com/goreleaser/nfpm => github.com/goreleaser/nfpm v1.10.2
 
-replace github.com/ory/kratos-client-go => github.com/ory/kratos-client-go v0.5.4-alpha.1.0.20210210170256-960b093d8bf9
+replace github.com/ory/kratos-client-go => github.com/ory/kratos-client-go v0.6.3-alpha.1
```

Comment:
This is in`.bin` so that we don't have the dependency in the main module. But also I don't want to always wait for the download, when go can rebuild stuff just on changes... But if you think that makes more sense, please release https://github.com/ory/cli/pull/90 or can I just make a release?

Reply:
The touch command will remove any uneccessary downloads! I'll create a release immediately

Reply:
once you change the makefile it re-downloads the binary?

Reply:
Release is in the making!